docs/requirements/software-requirements.md

```diff
@@ -0,0 +1,234 @@
+# CLARIAH Technical Requirements for Software and Services
+
+## Introduction
+
+This document intends to specify minimal technical requirements for CLARIAH
+software and software services. It is aimed at developers and external
+evaluators. It provides a minimal list of specific and directly actionable criteria to hold the software
+against. The aim is to ensure a certain level of software quality and
+sustainability, and to lay out basic requiments needed for interoperability
+within the larger CLARIAH infrastructure.
+
```

Comment:
Software quality and sustainability, important as they are, are no goals in themselves. Perhaps we can relate these goals to the _use_ of software, just as we do for the infrastructure requirements:

```suggestion
Only software that is built and maintained well will get used and thus provide value.

```